tests/pypi-indexes/generate-indexes.py

```diff
@@ -0,0 +1,60 @@
+"""This is a little script to generate a custom pypi simple index from a directory of source packages."""
+
+from pathlib import Path
+import shutil
+from build import ProjectBuilder
+
+indexes_path = Path(__file__).parent
+
+index_html_template = """<!DOCTYPE html>
```

Comment:
```suggestion
index_html_template = """\
<!-- This file has been generated with `generate-indexes.py` --> 
<!DOCTYPE html>
```

When running this script again, this should make it obvious where these files come from